nss-cohort-40/banagzon-workforce-mgt-upper-class-twits
# Mm secondday

Built out view and template and urls for employee form.

Please delete options that are not relevant.


# Testing Instructions
use the URL path: http://localhost:8000/employees/employee_form after fetching this mm-secondday branch. The blank form should appear and allow you to add an employee.
# Checklist:
